exporter/datadogexporter/config.go

```diff
@@ -494,6 +392,91 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+var _ error = (*renameError)(nil)
+
+// renameError is an error related to a renamed setting.
+type renameError struct {
+	// oldName of the configuration option.
+	oldName string
+	// newName of the configuration option.
+	newName string
+	// oldRemovedIn is the version where the old config option will be removed.
+	oldRemovedIn string
+	// issueNumber on opentelemetry-collector-contrib for tracking
+	issueNumber uint
```

Comment:
If you make it a string and add the full link, it will be easier to open, rather than copying a number and pasting it into a custom URL.